apps/web/src/server/app/components/file-downloader.component.js

```diff
@@ -20,11 +20,15 @@ const getDocumentsDownload = async ({ params, session }, response) => {
 
 	const accessToken = await getActiveDirectoryAccessToken(session);
 
-	const { blobStorageContainer, blobStoragePath } = await getCaseDocumentationFileUrl(
+	const { blobStorageContainer, blobStoragePath } = await getCaseDocumentationFileInfo(
 		caseId,
 		fileGuid
 	);
 
+	if (!blobStorageContainer || !blobStoragePath) {
+		throw new Error('Blobstorage container or blobStorage path not found');
```

Comment:
not quite related to the task but I did some refactor on the documentationFileInfo function and I noticed that this possible (although quite rare) error wasn't handled